cachi2/core/package_managers/pip.py

```diff
@@ -25,7 +25,6 @@
     Pattern,
     Union,
     cast,
-    no_type_check,
```

Comment:
**Commit message:**

> ensuring completion of the task.

What task? If read the commit in the git history, then without the context of the previous patches or this PR one doesn't know what particular task you're talking about. It should be something along the lines of "Enable typing for the PipRequirement class" or something similar. This commit is also the one which should contain the `resolves` keyword (I know you have it linked in you PR description, but that shouldn't be enough, because you don't have the information in your local git log) only in the web UI.